source/_integrations/backup.markdown

```diff
@@ -72,15 +56,43 @@ automation:
+The {% term "Home Assistant Container" %} installation typically mounts this directory via `docker-compose.yml` or `docker run` to a directory of your choice.
 
-If you use Home Assistant Operating System or Home Assistant Supervised, [the restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).
+The steps on recovering from a backup depend on the installation type and use case. Follow one of these steps:
 
-</div>
+1. If you use {% term "Home Assistant Operating System" %} or {% term "Home Assistant Supervised" %}:
+   - The restore functionality is already built-in.
+   - Follow the steps described in [Restoring a backup](/common-tasks/os/#restoring-a-backup).
+2. If you use a {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %} installation:
+   - A Home Assistant backup is just a tar file of the `/config` directory, plus some metadata.
+   - To manually restore a backup, use the following command:
 
-Backups created via the **Backup** integration are located in your `/config/backups` directory. The Home Assistant Container installation will typically mount this directory via `docker-compose.yml` or `docker run` to a directory of your choice.  
-For Container and Core installations, there is currently no built-in way to restore a backup. However, a Home Assistant backup is just a tar file of the `/config` directory, plus some metadata. To manually restore a backup, you can use the following:
+     ```shell
+        tar -xOf <backup_tar_file> "./homeassistant.tar.gz" | tar --strip-components=1 -zxf - -C <restore_directory>
+     ```
 
-```shell
-tar -xOf <backup_tar_file> "./homeassistant.tar.gz" | tar --strip-components=1 -zxf - -C <restore_directory>
-```
+3. If you migrate from a {% term "Home Assistant Core" %} or {% term "Home Assistant Container" %} installation to {% term "Home Assistant Operating System" %}:
+   - You can use these backup files during [onboarding](/getting-started/onboarding/) to recover your installation from backup.
+
+## Manual configuration
+
+The backup integration is enabled by default. If you've disabled or removed the [`default_config:`](/integrations/default_config/) line from your configuration, the backup functionality won't be available. Follow these steps to re-enable the backup integration.
+
+1. In your [configuration file](/docs/configuration/), re-add the following entry:
+
+    ```yaml
+    # Example configuration.yaml entry
+    backup:
+    ```
+
+2. Go to {% my system_dashboard title="**Settings**" %} and in the top right corner, select the three dots menu and select **Restart** > **Restart Home Assistant**.
+   - **Result**: When it has started up again, you will find a new **Backup** entry under ({% my backup title="**Settings** > **System** > **Backups**" %}**).
+   - The backup files are stored in a new **backups** subdirectory in the root of your configuration directory, `/config/backups/`.
```

Comment:
```suggestion
```

While you _can_ add this yo your configuration, doing so does nothing as this integration is always loaded regardless.
https://github.com/home-assistant/core/blob/a213de3db5524b065d56f9ba7a9d7d187ec6176b/homeassistant/bootstrap.py#L171